node-require/calculate.js

```diff
@@ -0,0 +1,9 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
```

Comment:
It would be best to save your user input into variables to save a lot of typing and cleanup the code a bit. For example:

```JAVASCRIPT
const op = process.argv[3];
const num1 = Number(process.argv[2]);
const num2 = Number(process.argv[4]);
```